vela-templates/addons/observability-asset/template.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  annotations:
+    addons.oam.dev/description: Some nessecerry X-definition for observability addon
+  name: observability-asset
+  namespace: vela-system
+spec:
+  workflow:
+    steps:
+      - name: apply-ns
+        type: apply-component
+        properties:
+          component: ns-observability
+      - name: apply-resources
+        type: apply-remaining
+  components:
+    - name: ns-observability
+      type: raw
+      properties:
+        apiVersion: v1
+        kind: Namespace
+        metadata:
+          name: observability
+    - name: import-grafana-dashboard-traitdef
+      type: raw
+      properties:
+        apiVersion: core.oam.dev/v1beta1
+        kind: TraitDefinition
+        metadata:
+          annotations:
+            definition.oam.dev/description: "Import dashboards to Grafana"
+          name: import-grafana-dashboard
+          namespace: vela-system
+        spec:
+          schematic:
+            cue:
+              template: |
+                outputs: registerdatasource: {
+                	apiVersion: "grafana.extension.oam.dev/v1alpha1"
+                	kind:       "ImportDashboard"
+                	spec: {
+                		grafana: {
+                			service:                   parameter.grafanaServiceName
+                			namespace:                 parameter.grafanaServiceNamespace
+                			credentialSecret:          parameter.credentialSecret
+                			credentialSecretNamespace: parameter.credentialSecretNamespace
+                		}
+                		urls: parameter.urls
+                	}
+                }
+                parameter: {
+                	grafanaServiceName:        string
+                	grafanaServiceNamespace:   *"default" | string
+                	credentialSecret:          string
+                	credentialSecretNamespace: *"default" | string
+                	urls: [...string]
+                }
+    - name: pure-ingress-traitdef
```

Comment:
Why not use our ingress trait instead？

Reply:
@zzxwill  why? 😂

Reply:
Now, at least this is different with our ingress trait ( remove isHealth)

Reply:
I need an Ingress who won't generate a service. Is there any? @FogDong

Reply:
Now our ingress will always generate a service.

Reply:
Thanks for your confirmation:)